### ISSUE TITLE
Scroll of ageless Wisdom Upgreade

### ISSUE THREAD

**Issue by HeavyDeception** [Extegral/Enigmatic-Legacy#212]
**Provide a detailed and clear description of (the) feature or change that you want to offer.**

Would it be possible to have an upgrade that allows the scroll to repair items with the mending enchant on them?

Also, One that allows you to open a GUI and make it so that the scroll keeps a set amount of XP on you at all times for doing mass enchants or repairs or if way stones is installed and you use the XP to travel places. Or it removes it from the scroll never seen that before but I don't know how balanced that is.

[This is more of a Quality of life change but could the XP be removed faster and by level and not amount because when you have a mod like Apotheosis it takes a lot of time for 150 levels to charge or to remove]

**Describe why you want this. If there's a reason.**
The upgrades are just ideas that came to mind when using this mod with other mods.

**Comment by Aizistral** [comment]
I actually like some of these ideas. Will see what I can do.

**Comment by Frostylooms** [comment]
i actually came here for a issue. the scroll wont stop giving me xp. but anyway i saw this and iv had a suggestion i bet a lot of people do. how about a 3rd mode like pause. pause the xp from going in or out. currently the only way to do this is to remove it from your slot while its on the output mode.

**Comment by Aizistral** [comment]
> how about a 3rd mode like pause

That third mode is called "disabled", and you can get it by either using Shift+Right-Click while holding scroll in hand or Shift+<scroll_specific_keybind_dunno_what_you_have_it_set_to> when equipped.

**Comment by Frostylooms** [comment]
> > how about a 3rd mode like pause
> 
> That third mode is called "disabled", and you can get it by either using Shift+Right-Click while holding scroll in hand or Shift+<scroll_specific_keybind_dunno_what_you_have_it_set_to> when equipped.

ahaha yes it is!. thank you. well disregard that part of my comment. any idea about why it might be giving me xp non stop while its active and in the slot?

**Comment by Aizistral** [comment]
> ahaha yes it is!. thank you. well disregard that part of my comment. any idea about why it might be giving me xp non stop while its active and in the slot?

'Cause... that's what it does? It either constantly drains your XP, constantly gives it back to you, or is disabled and does nothing. Ideally I would want there to be a GUI which let's you get your XP back in fixed portions, but I couldn't make it back in the day.

**Comment by HeavyDeception** [comment]
> > > how about a 3rd mode like pause
> > 
> > 
> > That third mode is called "disabled", and you can get it by either using Shift+Right-Click while holding scroll in hand or Shift+<scroll_specific_keybind_dunno_what_you_have_it_set_to> when equipped.
> 
> ahaha yes it is!. thank you. well disregard that part of my comment. any idea about why it might be giving me xp non stop while its active and in the slot?

Do you mean that it's empty but still giving you xp? Hover your mouse over it to see how much xp is in it when you have it turned.

**Comment by Frostylooms** [comment]
> > > > how about a 3rd mode like pause
> > > 
> > > 
> > > That third mode is called "disabled", and you can get it by either using Shift+Right-Click while holding scroll in hand or Shift+<scroll_specific_keybind_dunno_what_you_have_it_set_to> when equipped.
> > 
> > 
> > ahaha yes it is!. thank you. well disregard that part of my comment. any idea about why it might be giving me xp non stop while its active and in the slot?
> 
> Do you mean that it's empty but still giving you xp? Hover your mouse over it to see how much xp is in it when you have it turned.

yes, if hovering mouse over you can see the xp amount and levels  going up indefinitely while standing still doing nothing. i put the mod in the mod pack medieval Minecraft auto installed with cursefordge app on windows 10. it works just fine in better Minecraft mod pack. for references.

**Comment by Aizistral** [comment]
> yes, if hovering mouse over you can see the xp amount and levels going up indefinitely while standing still doing nothing. i put the mod in the mod pack medieval Minecraft auto installed with cursefordge app on windows 10. it works just fine in better Minecraft mod pack. for references.

Welp, if it's a modpack-specific issue - I can't help unless I know what particular mod(s) in that modpack causes this to happen. If you are willing to go through your modpack splitting it to eventually narrow down the culprit - then just do it and report a bug through proper form.
